app/assets/javascripts/social_share_button.js

```diff
@@ -0,0 +1,27 @@
+// Opening pop-ups with share URL
+function openShareUrl(url, initialWidth = 640, initialHeight = 480) {
+  if (typeof url !== "string" || !url.startsWith("http")) {
+    console.error("Invalid URL"); // Consider removing this line if console warnings should be avoided.
+    return;
+  }
+
+  const width = Math.max(100, Math.min(screen.width, initialWidth));
+  const height = Math.max(100, Math.min(screen.height, initialHeight));
+
+  const left = (screen.width / 2) - (width / 2);
+  const top = (screen.height * 0.3) - (height / 2);
+  const opts = `width=${width},height=${height},left=${left},top=${top},menubar=no,status=no,location=no`;
+
+  window.open(url, "popup", opts);
```

Comment:
Email link only works as expected for me when it's *not* opened with `window.open`.

It works when middle-clicked, or when the event handler is broken because of Turbo + `$(document).ready`.

Reply:
Still opens a window for `mailto:` links. What do you expect to appear in that window?

Reply:
Expected behaviour is to open mailto link in new tab where you can select your mail app/provider, which should be working for the timebeing. Do you suggest we should handle mail sharing differently?

Reply:
But you're also opening a new window that's not going to contain anything and will have to be closed manually. Or is it working differently for you?

Reply:
Actually for me it works like this - I'm using firefox, hadn't check for the other browsers.

When I click share it opens share link in pop up window to select the email provider:
<img width="1213" alt="Screenshot 2024-10-22 at 10 07 59" src="https://github.com/user-attachments/assets/d7eee376-638c-4684-ae79-96efecffd0b5">
After email provider selection, i get the automatically formatted email message ready to be sent:
<img width="886" alt="Screenshot 2024-10-22 at 10 08 23" src="https://github.com/user-attachments/assets/804f34bb-2ef3-43af-a615-df2b5bebd23f">

Reply:
Here's how it looks for me:
![image](https://github.com/user-attachments/assets/b84a3fbc-b52a-4223-aef3-5f09a19e218d)

I get a blank window in background and a new tab.

Reply:
In this case the email dialog gets opened right away. Fixed now, thanks.